engine/installation/linux/centos.md

```diff
@@ -79,16 +79,16 @@ Docker from the repository.
     > which will almost certainly be an unstable one.
 
     ```bash
-    $ sudo yum-config-manager --set-enabled docker-testing
+    $ sudo yum-config-manager --enable docker-testing
     ```
```

Comment:
This is right syntax on CentOS. '--set-enabled' does not work ('no such option).